experiments/customui/parent.js

```diff
@@ -273,7 +298,7 @@ var ex_customui = class extends ExtensionCommon.ExtensionAPI {
     // insertWebextFrame(). To remove frames created by this method, use
     // removeSidebarWebextFrame().
     const insertSidebarWebextFrame = function(location, url, document,
-        containerId) {
+        containerId, options) {
```

Comment:
Please adjust the comment describing a method if you change its signature.

Reply:
done